GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -93,7 +90,7 @@ typedef void (^GIDDisconnectCallback)(NSError *_Nullable error);
 ///
 /// @param callback The `GIDSignInCallback` block that is called on completion.  This block will be
 ///     called asynchronously on the main queue.
-- (void)restorePreviousSignInWithCallback:(nullable GIDSignInCallback)callback;
+- (void)restorePreviousSignInWithCallback:(nullable void (^)(GIDUserAuth *_Nullable userAuth, NSError *_Nullable error))callback;
```

Comment:
Nit: I think it's nicer to use `nullable` (and `nonnull`) than the underscored versions.

https://developer.apple.com/swift/blog/?id=25

Looks like both are used in this repo; we should probably settle on a style.

Reply:
Yes, nullable and nonnull are not working in block parameter.
https://stackoverflow.com/questions/29647919/how-to-use-nonnull-and-nullable-objective-c-keywords-in-block-based-api-method